parasite/scripts/production/parasite494/main.py

```diff
@@ -31,6 +31,8 @@
 
 # variable to hold term description so it can be configured easily
 gene_attrib_type = 'description'
+attrib_id = '49'
```

Comment:
Sorry, I meant the attrib_type_id you're getting in line 70.

Reply:
it comes up with an error as attrib_type_id isn't defined yet

Reply:
I'll rearrange the code so it creates the two queries after attrib_type_id is defined